core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -120,11 +122,11 @@ public static FileInStream create(URIStatus status, InStreamOptions options) {
    * @param status the file status
    * @param options the client options
    */
-  protected FileInStream(URIStatus status, InStreamOptions options) {
+  protected FileInStream(URIStatus status, InStreamOptions options, FileSystemContext context) {
```

Comment:
Yes, all contexts will be first. I'm doing a followup PR to make this consistent everywhere